scss/templates/user.scss

```diff
@@ -283,3 +283,67 @@
     margin-right: 0;
   }
 }
+
+// User Badges
+.user-content.user-badges-list {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(18.75rem, 1fr));
+  grid-gap: 1.6875rem;
```

Comment:
Why this is not `$grid-gutter-width` ? Be careful between pixel perfect and reproduce the same mistake that the designer potentially can do when creating the idea

Reply:
Fixed @duranmla